pkg/inventory/model/journal.go

```diff
@@ -61,6 +63,8 @@ type Watch struct {
 	journal *Journal
 	// Started
 	started bool
+	// Done
+	done bool
```

Comment:
Add state to track watch has ended so that _watcher_ can determine of a watch is still _alive_ or not.